sparse_strips/vello_common/src/strip.rs

```diff
@@ -83,23 +83,17 @@ pub fn render(
             macro_rules! fill {
                 ($rule:expr) => {
                     for x in 0..Tile::WIDTH as usize {
-                        let mut alphas = 0_u32;
-
                         for y in 0..Tile::HEIGHT as usize {
                             let area = location_winding[x][y];
                             let coverage = $rule(area);
-                            let area_u8 = (coverage * 255.0 + 0.5) as u32;
-
-                            alphas += area_u8 << (y * 8);
+                            alpha_buf.push((coverage * 255.0 + 0.5) as u8);
                         }
-
-                        alpha_buf.push(alphas);
                     }
                 };
             }
             match fill_rule {
                 Fill::NonZero => {
-                    fill!(|area: f32| area.abs().min(1.0))
+                    fill!(|area: f32| area.abs())
```

Comment:
The winding is now cast directly to `u8`, which saturates, so this `min` is unnecessary. Dropping this reduces strip rendering time by about `0.5ms` by itself.